accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/generators/monotonic/MonotonicAccessionGeneratorTest.java

```diff
@@ -400,4 +394,47 @@ public void postSaveAction() throws Exception {
         assertEquals(BLOCK_SIZE,accessions[2]);
     }
 
+    @Test
+    public void assertReleaseInAlternateRanges() throws Exception {
+        MonotonicAccessionGenerator generator = getMonotonicAccessionGenerator2();
```

Comment:
Does this really need to be a different generator? The method below uses `getMonotonicAccessionGenerator2` again so that is why I'm wondering.

Reply:
The generator2 is initialised with different category to test Alternate ranges , lemme have a generic function by passing category id then.